src/dotnet/commands/dotnet-new/Program.cs

```diff
@@ -88,8 +88,18 @@ public static int Run(string[] args)
             var dotnetNew = new NewCommand();
             app.OnExecute(() => {
 
-                var csharp = new { Name = "C#", Alias = new[] { "c#", "cs", "csharp" }, TemplatePrefix = "CSharp", Templates = new[] { "Console" } };
-                var fsharp = new { Name = "F#", Alias = new[] { "f#", "fs", "fsharp" }, TemplatePrefix = "FSharp", Templates = new[] { "Console" } };
+                var csharp = new { 
```

Comment:
it also add `"Lib", "xunittest"` to `Templates = new[] { "Console", "Lib", "xunittest" }`